src/modules/Elsa.Workflows.Core/Activities/Correlate.cs

```diff
@@ -0,0 +1,27 @@
+using System.ComponentModel;
+using System.Text.Json.Serialization;
+using Elsa.Workflows.Core.Attributes;
+using Elsa.Workflows.Core.Models;
+
+namespace Elsa.Workflows.Core.Activities;
+
+[Activity("Elsa", "Control Flow", "Set the CorrelationId of the workflow to a given value.")]
```

Comment:
Maybe change the category from "Control Flow" to "Primitives", what do you think?

Reply:
Good point. Makes sense therefore I updated